data/wanderer/wanderers middle.txt

```diff
@@ -20,6 +20,394 @@ mission "Wanderers: Pre-Colony"
+
+	on decline
+		event "wanderers: spera anatrusk colony" 20
+
+	on abort
+		event "wanderers: spera anatrusk colony" 20
+
+	on fail
+		event "wanderers: spera anatrusk colony" 20
+
+	on stopover
+		dialog "The dock workers seem eager to load the massive freighters. They are excited to be part of the Migration. They stand together and salute the flight crew when they board for departure."
+	
+	on visit
+		dialog "You've arrived at <planet> but not all the freighters have landed. You realize you need to depart and make sure they arrive safely."
+
+	npc accompany
+		planet "Vara K'chrai"
+		fleet 
+			names "wanderer"
+			cargo 5
+			outfitters "Wanderer Basics"
+			variant
+				"Deep River"
+				"Deep River"
+				"Deep River"
+	
+	on complete
+		dialog "Once the Thunderhead missiles start unloading, you see a flurry of Wanderers as captains and crews fly over and besiege the outfitter staff, wanting to know when they can place an order. The construction supervisior comes over and moves his head in a way you've come to associate with humor as he watches the commotion at the other landing pad. He hands you a credit chip for <payment>."
+		payment 3000000
+		event "wanderers: spera anatrusk outfitter basic"
+
+
+mission "Wanderers: Spera Anatrusk Outfitter 5"
+	landing
+	name "Escort outfitter convoy"
+	description "Escort outfitter convoy to <destination>."
+	blocked "You are [lacking, missing] <capacity> to transport these Wanderer leaders."
+	source "Spera Anatrusk"
+	destination "Spera Anatrusk"
+	stopover "Vara K'chrai"
+	passengers 30
+
+	to offer
+		has "Wanderers: Spera Anatrusk Outfitter 4: done"
+		
+	on offer
+		conversation
+			`The spaceport building is taking shape, with all the foundations set and the walls of the main hall in place. It probably helps that the Wanderers are able to fly about all levels of the structure without needing to traverse ladders, but it's more their teamwork and dedication at work. It's like a human barn raising but on a larger scale.`
+			`	The manager you brought over a few weeks ago to open the outfitter flaps down and lands beside you. You both watch the construction for a moment and then she says, "The [Leader, Director] of Spaceports has said that we must have a full outfitter here to support the [Trek, Migration]. We are sending another convoy to pick up the more advanced outfits and some dignitaries to officially [inaugurate, open] the outfitter. They said it would be [appropriate, fitting] for you to escort them.`
+			
+			choice
+				`	"I'd be honored."`
+					accept
+				`	"It's too dangerous to transport Wanderer dignitaries here."`
+					decline
+			`	"You are much like us," she says. "You carry seeds from place to place and some take root and grow and over thousands of years transform worlds.`
```

Comment:
```suggestion
			`	"You are [much like], [similar to] us," she says. "You carry seeds from place to place and some take root and grow and over thousands of years they transform worlds.`
```

Reply:
I accept the suggestion but will revise the format to [much like, similar to] . All the examples I can find have all options within the brackets.